comparison.md

```diff
@@ -76,7 +76,7 @@ Help improve our documentation! If you find anything wrong or which could be imp
 				<td><strong>Locations</strong></td>
 				<td>Anywhere in the world possible, existing servers in: Switzerland, USA, South Africa, UK</td>
 				<td>US only</td>
-				<td>Bound to Amazon AWS: US, Ireland, Japan, Singapore, Australia</td>
+				<td>Bound to Amazon AWS: US, Ireland, Frankfurt, Japan, Singapore, Brazil, Australia</td>
```

Comment:
I think this should also read "US (East and West)", as they are two separate datacenters.